src/actinia_core/resources/raster_export.py

```diff
@@ -221,8 +221,13 @@ def _execute(self):
         self.required_mapsets.append(self.mapset_name)
         self._create_temporary_grass_environment(source_mapset_name="PERMANENT")
 
+        format = "COG"
```

Comment:
```suggestion
        # COG bug in GDAL, see https://github.com/OSGeo/gdal/issues/2946 will be fixed in GDAL 3.1.4
        # use r.out.gdal -c to avoid the bug
        format = "COG"
```